### ISSUE TITLE
How to label the images?

### ISSUE THREAD

**Issue by ggsf13** [sebastianbk/finetuned-resnet50-keras#4]
I need to do a binary classification task, should I just name the folders in the train and valid folders by the name of the classes? 
And could you tell me if I can use this to train my own dataset that is beyond the 1000 classes?
